libraries/tensorflow-gpu-1.12/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# Install the requirements defined in the requirements.txt file in the template
+pip install -r ../../templates/tensorflow-gpu-1.12/requirements.txt
+
+# Give algo user ability to write to updated keras/tensorflow files
+find $PYTHON_LIB_PATH/$PYTHON_VERSION/site-packages -user root | xargs chown algo:algo
```

Comment:
This doesn't seem ideal, what's the use case?

Reply:
The pip installation above is done under root.

This would prevent a user to update their `tensorflow-gpu` version if they desire to do so.

Reply:
They shouldnt be using this image if they wanted a different version of tensorflow though, that would be a big waste of space. They could just use the normal pythin+cuda image. I'm also not certain those variables would be set.

Reply:
yeah, arguably that would defeat the purpose of the packageset. We should just have package sets per TF version

Reply:
This is with the assumption that we won't be able to change an algorithms packageset retroactively, otherwise the user would have to create a new algorithm from scratch.

But I guess that's an issue out-of-scope here.

I'll remove the additional command.

Reply:
Users will be able to change the package set (will take effect on next compile)

Reply:
Good to know.